minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -843,7 +843,7 @@ public synchronized Properties getClientProperties() {
       clientProperties =
           Accumulo.newClientProperties().from(config.getClientPropsFile().toPath()).build();
     }
-    return clientProperties;
+    return new Properties(clientProperties);
```

Comment:
This change is causing MiniAccumuloCluster tests to fail, the error says that the `instance.name` is not set. The properties must be modified after returning the reference. Reverting this change allows the tests to pass.

Reply:
Chatted w/ @dlmarion  and he suggested this was a pre-existing issue and that we should not address it in this branch. 
 So I opened apache/accumulo#2619